support/mender-inventory-geo

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# Example script called by Mender agent to collect inventory data for a
```

Comment:
`s/Mender agent/Mender client/`

Reply:
https://github.com/mendersoftware/mender/blob/master/support/mender-device-identity#L3
https://github.com/mendersoftware/mender/blob/master/support/mender-inventory-network#L3
do I correct all other places as well?

Reply:
A separate PR would be nice I think :)

Reply:
https://github.com/mendersoftware/mender/pull/544